tests/check_sssd_with_tlog.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Test should be a playbook, and you have a list of tasks. Check https://github.com/linux-system-roles/tlog/blob/main/tests/tests_cockpit.yml

Reply:
Fixed.

Reply:
Test playbook filenames must match named `tests_*.yml`
I thought perhaps that `check_sssd_with_tlog.yml` was going to be a task file called from other playbooks . . .

Reply:
Ahh okay, I see now. I fixed it so that `check_sssd_with_tlog` is called from `tests/tests_sssd.yml`